server/src/main/java/server/haengdong/application/response/ActionAppResponse.java

```diff
@@ -0,0 +1,52 @@
+package server.haengdong.application.response;
+
+import server.haengdong.domain.action.BillAction;
+import server.haengdong.domain.action.MemberAction;
+import server.haengdong.domain.action.MemberActionStatus;
+
+public record ActionAppResponse(
+        Long actionId,
+        String name,
+        Long price,
+        Long sequence,
+        ActionType actionType
+) {
+
+    public static ActionAppResponse of(BillAction billAction) {
+        return new ActionAppResponse(billAction.getAction().getId(),
+                billAction.getTitle(),
+                billAction.getPrice(),
+                billAction.getSequence(),
+                ActionType.BILL
+        );
+    }
+
+    public static ActionAppResponse of(MemberAction memberAction) {
+        MemberActionStatus status = memberAction.getStatus();
+
+        return new ActionAppResponse(memberAction.getAction().getId(),
+                memberAction.getMemberName(),
+                null,
+                memberAction.getSequence(),
+                ActionType.of(status)
+        );
+    }
+
+    public String actionTypeName() {
+        return actionType.name();
+    }
+
+    private enum ActionType {
+        BILL,
+        IN,
+        OUT,
+        ;
+
+        public static ActionType of(MemberActionStatus memberActionStatus) {
+            if (memberActionStatus.name().equals("IN")) {
```

Comment:
```suggestion
            if (memberActionStatus == MemberActionStatus.IN) {
```
r: 그럴 가능성이 낮긴 하지만 MemberActionStatus.IN의 이름이 변경될 경우에 단축키로 한 번에 변경할 수 있으니 문자열보다는 enum으로 직접 비교하는 것이 더 좋을 것 같아요~